taf/developer_tool.py

```diff
@@ -843,6 +843,7 @@ def generate_repositories_json(
     namespace=None,
     targets_relative_dir=None,
     custom_data=None,
+    use_mirrors=True,
```

Comment:
Could you update docstring? Also, line 861. has typo: `custom_date` -> `custom_data`